### PR TITLE
Add IDE settings and update README.md

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Add common IDE code style and import definitions and update `README.md` with installation instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,39 @@ Please read Admin and User manuals in doc/ folder
...
+* Select *Settings* in the *File* menu
+* Select *Editor*
+* Select *Code Style*
+* Expand the menu item and select *Java*
+* Go to *Scheme* at the top, click on the setting button by the side of the drop-down list
+* Select *Import Scheme*
+* Select *IntelliJ IDE code style XML*
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/idea/OpenHospital-code-style-configuration.xml`
+* Select *OK* 
+* At this point the code style is stored as part of the IDE and is used for **all** projects opened in the editor.  To restrict the settings to just this project again select the setting button by the side of the *Scheme* list and select *Copy to Project...*. If successful a notice appears in the window that reads: *For current project*.
```
Comment (mwithi) [inline README.md:56]: I would suggest a collapsable section for the list, like this:

<details>
  <summary>IntelliJ instructions</summary>

* list1
* list2

</details>

File: README.md [hunk 2]
```diff
@@ -38,6 +38,39 @@ Please read Admin and User manuals in doc/ folder
...
+* Select *Formatter*
+* Select the *Import...* button
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/eclipse/OpenHospital-Java-CodeStyle-Formatter.xml`
+* Select *Open*
+* At this point the code style is stored and is applicable to all projects opened in the IDE.  To restrict the settings just to this project select *Configure Project Specific Settings...* in the upper right.  In the next dialog select the *openhospital* repository and select *OK*.  In the next dialog select the *Enable project specific settings* checkbox.  Finally select *Apply and Close*.
+* Back in the *Code Style* menu area, select *Organize Imports*
+* Select *Import...*
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/eclipse/OpenHospital.importorder`
+* Select *Open*
+* As with the formatting styles the import order is applicable to all projects.  In order to change it just for this project repeat the same steps as above for *Configure Project Specific Settings...*
```
Comment (mwithi) [inline README.md:72]: I would suggest a collapsable section for the list, like this:

<details>
  <summary>Eclipse instructions</summary>

* list1
* list2

</details>

**Review by mwithi — APPROVED** [review]
For me is ok, if you like we can add a collapsable section for each instructions list

File: README.md (hunk 2 above)
Comment (dbmalkovsky) [reply README.md:78]: Done for both IDEs

**Pull request merged**